PowerShell/README.md

```diff
@@ -0,0 +1,52 @@
+# BitTitan PowerShell
+This folder contains a set of tools, building blocks and full scenario scripts tailored to work with the BitTitan PowerShell SDK. 
+
+## Quick Startup
+1. Install the [BitTitan PowerShell SDK](https://www.bittitan.com/downloads/bittitanpowershellsetup.msi).
+2. Clone the [bittitan-sdk repository](https://github.com/BitTitan/bittitan-sdk) or download the desired scripts.
+    ```shell
+    git clone https://github.com/BitTitan/bittitan-sdk.git
+    ```
+3. Move the `Init.ps1` (bittitan-sdk/PowerShell/Tools/Init.ps1) script to the working directory. If the repository was not cloned, download the [Init script](https://github.com/BitTitan/bittitan-sdk/blob/master/PowerShell/Tools/Init.ps1) and move it to the working directory.
+4. Execute desired scripts from the working directory.
+
+## Contents
+* `/BuildingBlocks (Building Blocks)` - Samples to help developers build scripts that automate various operations.
+* `/Support (Support Scripts)` - Scripts that enable developers address numerous scenarios by utilizing them.
```

Comment:
I would say: "enable developers address specific scenarios or self-troubleshoot issues".
The idea is to explain how these are different from the BuildingBlocks scripts

Reply:
Good idea! I've changed that, thanks :)